frontend/src/component/common/ConstraintAccordion/ConstraintIcon.tsx

```diff
@@ -12,6 +12,7 @@ export const ConstraintIcon: VFC<IConstraintIconProps> = ({
     disabled,
 }) => (
     <Box
+        className='constraint-icon-container'
```

Comment:
Allow us to target ConstraintIcon with classes in styled component

Reply:
I haven't seen this in other components. Is it the only option to achieve what we want?

Reply:
Instead of drilling props down to this component, we give it an identifier that we can use in css. It's a quite common pattern that we don't use that much due to mostly handling it with styled components. The benefit of this is that it will work with the new constraint work that we have done aswell, otherwise we'd need to drill the props through two sets of components. ConstraintAccordionList and NewConstraintAccordionList (and it's associated copied components).